### PR TITLE
Feat/pagination first & last button

### LINKED ISSUE #194: Pagination changes
- Fast forward to the last page button
- Fast forward to first-page button

### PR DESCRIPTION
**Pull request by Codered9**
# Description
- Fast forward to the last page button
- Fast forward to first-page button

Fixes #194 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

# Screenshot of new Pagination:
image

### CODE REVIEW

**Review by shvam0000 — APPROVED** [review]
👍🏻

File: client/client-portals/components/admin/dashboard/accepted-projects/index.tsx [hunk 1]
```diff
@@ -62,6 +64,7 @@ const AcceptedProjectDashboard = () => {
                 hasPrevPage={hasPrevPage}
                 pageNo={pageNo}
                 setPageNo={setPageNo}
+                lastPage={lastPage}
```
Comment (jagnani73) [inline client/client-portals/components/admin/dashboard/accepted-projects/index.tsx:67]: Use spread operator if all the keys are same as propNames

File: client/client-portals/components/maintainer/dashboard/dashboard.tsx [hunk 2]
```diff
@@ -74,6 +76,7 @@ const index = () => {
               hasPrevPage={hasPrevPage}
               pageNo={pageNo}
               setPageNo={setPageNo}
+              lastPage={lastPage}
```
Comment (jagnani73) [inline client/client-portals/components/maintainer/dashboard/dashboard.tsx:79]: Use spread operator if all the keys are same as propNames

File: client/client-portals/utils/icons/last.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const Last = () => (
+    <svg stroke="currentColor"
+      fill="currentColor"
+      strokeWidth="0"
+      viewBox="0 0 1024 1024"
+      height="1em"
+      width="1em"
```
Comment (jagnani73) [inline client/client-portals/utils/icons/last.tsx:7]: replace with `className="h-full w-full"`

**Review by jagnani73 — CHANGES_REQUESTED** [review]

File: client/client-portals/utils/icons/last.tsx (hunk 3 above)
Comment (Codered9) [reply client/client-portals/utils/icons/last.tsx:7]: I have addressed this.

File: client/client-portals/components/admin/dashboard/accepted-projects/index.tsx (hunk 1 above)
Comment (Codered9) [reply client/client-portals/components/admin/dashboard/accepted-projects/index.tsx:67]: They are separate values. Should I create an array of all those values then use the spread operator on it?

**Review by sakshi-choudhary — APPROVED** [review]

**Review by shvam0000 — APPROVED** [review]
👍🏻

**Pull request merged**